### PR TITLE
Feat: Cart Page

### PR DESCRIPTION
**Pull request by Muzammil-khan710**
Added add to cart and Remove from cart functionality
This works on both product page as well as cart listing page

Not to check:
filters
quantity on cart listing page
available categories on Home page

### CODE REVIEW

File: src/Components/Cart/CartPage.css [hunk 1]
```diff
@@ -0,0 +1,76 @@
+.horizontal-card {
+    display: flex;
+    border: 2px solid;
+    justify-content: space-between;
+    width: 35rem
+}
+
+.horizontal-card-img {
+    height: 200px;
+    width: 250px;
```
Comment (Shubhamkr2610) [inline src/Components/Cart/CartPage.css]: use rem here

File: src/Components/Products/ProductsPage.jsx [hunk 2]
```diff
@@ -65,12 +71,46 @@ const ProductsPage = () => {
...
+                    <button className="btn red" onClick={() => {
+                      dispatch({
+                        type: "REMOVE_FROM_CART",
+                        paylaod: item
+                      })
+                    }}>
                       Move to Wishlist
-                    </button>
+                    </button> */}
+
```
Comment (Shubhamkr2610) [inline src/Components/Products/ProductsPage.jsx]: remove unused code

**Review by Shubhamkr2610 — APPROVED** [review]
looks good to merge

File: src/Components/Products/ProductsPage.jsx [hunk 3]
```diff
@@ -3,9 +3,15 @@ import "./ProductPage.css";
...
 import React from 'react'
 import { IcBaselineStar } from '../../images/Svg';
+import { useCart } from '../../context/Cart-context';
 
 const ProductsPage = () => {
   
+  const { state: {cart}, dispatch } =  useCart()
+
+
+  console.log(cart)
```
Comment (zoyatanweer) [inline src/Components/Products/ProductsPage.jsx]: remove console.log

**Review by zoyatanweer — APPROVED** [review]
clean code, good to merge

**Pull request merged**